ticdc/ticdc-csv.md

```diff
@@ -23,7 +23,7 @@ protocol = "csv"
 terminator = "\n"
 
 [sink.csv]
-delimiter = ','
+delimiter = ',' # Before v7.6.0, you can only set the delimiter to a single character. Starting from v7.6.0, you can set it to 1~3 characters. For example, `$^` or `|@|`.
```

Comment:
Instead of `to 1~3 characters` it should be `to 1-3 characters`.

The `-` chatacter is used when writing a range of numbers, as in this example. The `~` character, or tilde, is used when writing an approximate number, e.g. `approximately 10` would be written as `~10`.